src/client/pkg/pbutil/pbutil.go

```diff
@@ -65,7 +66,7 @@ func (r *readWriter) WriteBytes(bytes []byte) (int64, error) {
 	}
 	lenByteSize := unsafe.Sizeof(int64(len(bytes)))
 	n, err := r.w.Write(bytes)
-	return int64(lenByteSize) + int64(n), err
+	return int64(lenByteSize) + int64(n), errors.WithStack(err)
```

Comment:
I don't think we want this here. This should not be the line between a vendor's code and ours. Some of the places above that use the binary / proto package seem like better candidates.